rest/route/version_create.go

```diff
@@ -66,24 +62,18 @@ func (h *versionCreateHandler) Run(ctx context.Context) gimlet.Responder {
 	}
 	projectInfo.IntermediateProject, err = model.LoadProjectInto(ctx, h.Config, opts, projectInfo.Ref.Id, p)
 	if err != nil {
-		return gimlet.NewJSONErrorResponse(gimlet.ErrorResponse{
-			StatusCode: http.StatusBadRequest,
-			Message:    errors.Wrap(err, "unable to unmarshal yaml config").Error(),
-		})
+		return gimlet.NewJSONInternalErrorResponse(errors.Wrap(err, "loading project from config"))
 	}
 	if projectInfo.Ref.IsVersionControlEnabled() {
 		projectInfo.Config, err = model.CreateProjectConfig(h.Config, projectInfo.Ref.Id)
 		if err != nil {
-			return gimlet.NewJSONErrorResponse(gimlet.ErrorResponse{
-				StatusCode: http.StatusBadRequest,
-				Message:    errors.Wrap(err, "unable to unmarshal project config yaml").Error(),
-			})
+			return gimlet.NewJSONErrorResponse(errors.Wrap(err, "creating project config"))
```

Comment:
500 and `creating config for project '%s'`

Reply:
Done.